apphelpers/utilities/email.py

```diff
@@ -90,6 +92,29 @@ def send_email(
     """
     assert any((text, html)), "please provide html or text"
 
+    if settings.DEBUG or settings.APP_MPDE != "prod":
```

Comment:
Suggestion: we should also filter bcc list, 
another suggestion: Maybe this code is better suited at app level, as it is specific to our use case.

Reply:
bcc +1
The use case argument applies to the whole repo though.